packages/js/product-editor/src/hooks/use-product-variations-helper.ts

```diff
@@ -9,13 +9,50 @@ import {
 	EXPERIMENTAL_PRODUCT_VARIATIONS_STORE_NAME,
 	Product,
 	ProductDefaultAttribute,
+	ProductVariation,
 } from '@woocommerce/data';
 
 /**
  * Internal dependencies
  */
 import { EnhancedProductAttribute } from './use-product-attributes';
 
+async function getDefaultVariationValues(
+	productId: number
+): Promise< Partial< Omit< ProductVariation, 'id' > > > {
+	const attributes = (
```

Comment:
Also, if you are making a get product request in line 76 to get the status, why not getting the attributes there too, to avoid this request.

`76: const { status: lastStatus, attributes } = await ...`

Reply:
Aaah yeah, that looks much cleaner.
Btw I didn't do the merge of status and attributes, as the status is coming from the edited product, while I am getting attributes from un-edited product.
Although neither should be triggering a request ( I don't think ) as they should already be available through redux.